src/callbacks/findNearestDriverIndex.js

```diff
@@ -0,0 +1,17 @@
+function findNearestDriverIndexI(response) {
```

Comment:
as said in previous PR change to anonymous function, and you have typo in your name, last `I`